rmw_connext_cpp/src/functions.cpp

```diff
@@ -883,7 +883,7 @@ rmw_wait(rmw_subscriptions_t * subscriptions,
   rmw_guard_conditions_t * guard_conditions,
   rmw_services_t * services,
   rmw_clients_t * clients,
-  bool non_blocking)
+  rmw_time_t * wait_timeout)
```

Comment:
The function must check that the pointer is valid (if it stay a pointers).
(Actually it should also check all the other pointers coming in.)

Same below and for OpenSplice.